src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php

```diff
@@ -453,6 +427,36 @@ private function getCalculatedPrices($type = 'rent', $currency) {
 
                 $nullcheck = array_merge($nullcheck, $nullcheck_addition);
             }
+
+            $extraCosts = null;
+            if (isset($this->property['_embedded']['extracosts'])) {
+                foreach ($this->property['_embedded']['extracosts'] as $extracost) {
```

Comment:
Why move this down? :D

Reply:
@josias-r that it keeps the same order than it has now on iCasa :)